### ISSUE TITLE
[Bug] Position offset in PlainTextParser

### ISSUE THREAD

**Issue by takahi-i** [recruit-tech/redpen#437]
I got a report on a bug offset in JSON output. I will look into the problem.

**Comment by takahi-i** [comment]
When I input the following text.

```
ラ、、になる。ラ、、
になる。ラ、、になる。
```

I got the correct result with Markdown Parser.

```
{
    "errors": [
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 0,
                    "line": 1
                },
                "end": {
                    "offset": 6,
                    "line": 1
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 2,
                            "line": 1
                        },
                        "end": {
                            "offset": 3,
                            "line": 1
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        },
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 7,
                    "line": 1
                },
                "end": {
                    "offset": 3,
                    "line": 2
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 9,
                            "line": 1
                        },
                        "end": {
                            "offset": 0,
                            "line": 2
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        },
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 4,
                    "line": 2
                },
                "end": {
                    "offset": 10,
                    "line": 2
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 6,
                            "line": 2
                        },
                        "end": {
                            "offset": 7,
                            "line": 2
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        }
    ]
}
```

But unfortunately the line number is not correct with PlaintextParser.

The point is that the line number of second sentence. The start position should be the the following (line 38th).

```
            "position": {
                "start": {
                    "offset": 7,
                    "line": 1
                },
                "end": {
                    "offset": 3,
                    "line": 2
                }
            },
```

But the result with PlainTextParser is the follows.

```
            "position": {
                "start": {
                    "offset": 4,
                    "line": 2
                },
                "end": {
                    "offset": 10,
                    "line": 2
                }
            },
``` 


```
{
    "errors": [
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 0,
                    "line": 1
                },
                "end": {
                    "offset": 6,
                    "line": 1
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 2,
                            "line": 1
                        },
                        "end": {
                            "offset": 3,
                            "line": 1
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        },
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 4,
                    "line": 2
                },
                "end": {
                    "offset": 10,
                    "line": 2
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 6,
                            "line": 2
                        },
                        "end": {
                            "offset": 7,
                            "line": 2
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        },
        {
            "sentence": "ラ、、になる。",
            "position": {
                "start": {
                    "offset": 7,
                    "line": 2
                },
                "end": {
                    "offset": 3,
                    "line": 2
                }
            },
            "errors": [
                {
                    "subsentence": {
                        "offset": 2,
                        "length": 1
                    },
                    "validator": "SuccessiveWord",
                    "position": {
                        "start": {
                            "offset": 9,
                            "line": 1
                        },
                        "end": {
                            "offset": 0,
                            "line": 2
                        }
                    },
                    "message": "Found word \"、\" repeated twice in succession."
                }
            ]
        }
    ]
}
```

**Comment by gerryhocks** [comment]
I will have a look into this and fix the issue

**Comment by takahi-i** [comment]
Thank you very much for the contribution @gerryhocks! Let me know if you have any problem, especially the Japanese sample text.

**Comment by gerryhocks** [comment]
No problem! 

The sample Japanese text is quite readable so I should be okay.

**Comment by gerryhocks** [comment]
I forgot to mention in the AsciiDoc pull-request, but this should be fixed as part of those updates.

**Comment by takahi-i** [comment]
great!

**Comment by takahi-i** [comment]
Fixed with #444
